README.md

```diff
@@ -71,32 +73,23 @@ _Long-form documentation for this section can be found [here](docs/build_and_loa
 
 > [!NOTE]
 > This section is currently in progress and command input may be
-> subject to change. The following pseudocode is a possible
-> representation of the imputation workflow:
+> subject to change.
 
 ```shell
+## Export
+./phg export-vcf --db-path /my/db/uri --dataset-type hvcf --sample-names LineA,LineB --output-dir /my/hvcf/dir
+
 ## Index
-./phg index-kmers --ancestor founder.h.vcf -o kmer_index.map // we need this
+./phg build-kmer-index --db-path /my/db/uri --hvcf-dir /my/hvcf/dir
 
 ## Map
-./phg map-kmers \
-    --kmer-index kmer_index.map \
-    --reads my_reads.fastq \ // possibly thousands of samples being inputted
-    --output read_count_out.map \ // could we pipe this into impute method? // thousands of outputs
-    // consider batch interface here ^^
-
-## Impute
-./phg impute \
-    --hap-counts read_count_out.map \ // will users understand the di
-    --diploid false \
-    --ancestor founder.h.vcf \
-    --max-anc-hap-num 20 \
-    --max-anc-hap-prop 0.95 \
-    --output-parent best_parents.txt \
-    -o my_impute.h.vcf
-
-## Load
-./phg load-vcf --vcf my_impute.vcf --dbpath /my/db/uri
+./phg map-kmers --hvcf-dir /my/hvcf/dir --kmer-index /my/hvcf/dir/kmerIndex.txt --read-files /path/to/reads/LineA_R1.fq /path/to/reads/LineA_R2.fq --output-dir /my/mapping/dir
```

Comment:
Probably should change the option "--read-files /path/to/reads/LineA_R1.fq /path/to/reads/LineA_R2.fq" to "--key-file /my/path/keyfile". I think using a keyfile will be the usual way of running map-kmers.

Reply:
Modified